docs/error-reporting/platform-integrations/source-map.md

```diff
@@ -9,88 +9,259 @@ import Tabs from '@theme/Tabs';
+2. Add the following scripts to your `package.json` file:
 
 ```json
-"devDependencies": {
-  "backtrace-sourcemap-tools": "git@github.com:backtrace-labs/backtrace-sourcemap-tools.git"
+"scripts": {
+  "backtrace:process": "backtrace-js process OUTPUT_DIRECTORY",
+  "backtrace:upload": "backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
 }
 ```
 
-## Step 3: Configure Sourcemap Upload URL in `package.json`
-
-Specify the URL, token, and universe where Backtrace sourcemap tools will upload sourcemap files when your project is built. Add the following configuration to your `package.json` file under the top-level key `backtrace`:
+You can also use `npx` and skip adding the dependency:
 
 ```json
-"backtrace": {
-  "sourcemap": {
-    "upload": "https://<universe>.sp.backtrace.io:6098/post?format=sourcemap&token=<token>"
-  }
+"scripts": {
+  "backtrace:process": "npx --package @backtrace/javascript-cli backtrace-js process OUTPUT_DIRECTORY",
+  "backtrace:upload": "npx --package @backtrace/javascript-cli backtrace-js upload OUTPUT_DIRECTORY UPLOAD_OPTIONS"
 }
 ```
 
-You need to replace `<universe>` with your universe name, and `<token>` with the symbol upload token generated in your Project Settings under the Symbols section. For on-premise Backtrace servers, replace `<universe>.sp.backtrace.io:6098` with the server and port of your Backtrace instance.
+Ensure to replace `OUTPUT_DIRECTORY` with the path to the directory where your transpiled scripts are stored.
 
-## Step 4: Set Up Automatic Sourcemap Uploads
+Replace `UPLOAD_OPTIONS` with either `--subdomain <your subdomain> --token <your token>` or `--url <your upload URL>`, obtained from [the symbol submission URL](#obtaining-symbol-submission-url).
+
+#### Configuration File
+Instead of providing options in script argument lines, you can configure them in the `.backtraceclirc` configuration file:
+
+```jsonc
+{
+  "path": "OUTPUT_DIRECTORY",
+  "upload": {
+    "subdomain": "your subdomain",
+    "token": "your token",
+    // or
+    "url": "your upload URL"
+  }
+}
+```
 
-This method uploads all artifact .map files.
+For more details consult `@backtrace/javascript-cli` README.
 
-First, append `npm run upload` to the `build` command in your package.json's `scripts` section:
+### Step 3: Set Up Automatic Processing and Upload
 
+To process and upload your artifacts, you must first run `npm run backtrace:process` and then `npm run backtrace:upload`.
+
+To ensure that this is done automatically, you can add these commands to your production script:
 ```json
 "scripts": {
-  "build": "my current build command && npm run upload",
-  ...
+  "build": "my current build command && npm run backtrace:process && npm run backtrace:upload"
 }
 ```
 
-Finally, add the action script `upload` to run `backtrace-sourcemap` with `package.json` and the build folder as parameters. In this example, both the `package.json` and the `dist` folder are assumed to be located in the project root directory:
+## Webpack
```

Comment:
Webpack, Rollup and Vite are mutually exclusive. Can we create a "Project Bundler Instructions" section, and have these tabbed within?